resources/lib/method.py

```diff
@@ -152,6 +152,28 @@ def _get_preselected_items(string):
             return -1
         return items.index(string)
 
+    def _get_preselected_items(string):
+        if not string:
+            return [-1]
+        try:
+            return [int(string)]
+        except TypeError:
+            return [-1]
+        except ValueError:
+            pass
+        items = _split_items(kwargs.get('preselect') or '')
+        if not items:
+            return [-1]
+        if len(items) == 0:
+            return [-1]
+        selected = []
+        for i in items:
+            try:
+                selected.append(int(i))
```

Comment:
This is fine **if** ints are passed. However, the basic select function also allows passing strings to match the listitems so we need to look for the index too. I've got something for this incoming.